features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -78,31 +78,37 @@
 	</feature>
 
 	<feature name="openhab.tp-jackson" description="FasterXML Jackson bundles" version="${project.version}">
-		<capability>openhab.tp;feature=jackson;version=2.9.10</capability>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.datatype/jackson-datatype-joda/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.9.10</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.9.10</bundle>
-		<bundle dependency="true">mvn:org.yaml/snakeyaml/1.23</bundle>
+		<capability>openhab.tp;feature=jackson;version=2.10.3</capability>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.datatype/jackson-datatype-joda/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.10.3</bundle>
+		<bundle dependency="true">mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.10.3</bundle>
+		<bundle dependency="true">mvn:org.yaml/snakeyaml/1.26</bundle>
 	</feature>
 
-	<feature name="openhab.tp-jax-rs" version="${project.version}">
-		<capability>openhab.tp;feature=jax-rs;version=5.3.1</capability>
-		<feature>openhab.tp-jax-rs-min</feature>
-	</feature>
-
-	<feature name="openhab.tp-jax-rs-min" version="${project.version}">
```

Comment:
Can you explain this use case? The bundle will always be installed because it is a  dependency of [openhab-core-base](https://github.com/openhab/openhab-core/blob/4bdf8166e3ac0eb5ab4ccedd3cb0e8d4d5a3c7df/features/karaf/openhab-core/src/main/feature/feature.xml#L20-L32).